engine/src/flutter/ci/builders/linux_android_emulator.json

```diff
@@ -30,6 +30,8 @@
             "ninja": {
                 "config": "ci/android_emulator_debug_x64",
                 "targets": [
+                    "flutter/impeller/renderer/backend/vulkan:vulkan_android_unittests",
+                    "flutter/impeller/renderer/backend/vulkan:vulkan_android_apk_unittests",
```

Comment:
This is just building the tests right?  it isn't executing them.

Reply:
correct, but the tests are run as part of the existing android test shard as I've added them here: https://github.com/flutter/flutter/pull/164744/files#diff-208f1d19de16cc0af95bc6c1c374cde2f450d73098b217e66b3cf98f1b097b3cR767

Reply:
got it, awesome